packages/client-api-schema/src/types.ts

```diff
@@ -160,9 +160,18 @@ export interface SiteBudget {
   inUse: Balance;
   direct: Balance;
 }
+
+export interface BudgetRequest extends Balance {
+  site: string;
+  playerDestinationAddress: string;
+  hubAddress: string;
```

Comment:
Why is the hub necessarily attached to a `BudgetRequest`?

Reply:
What would be the case when there wouldn't be a hub? Are you suggesting the `wallet` gets to decide the `hub`?

Reply:
I don't think budgets are related to hubs

Reply:
An approved budget should result in a funded ledger channel with a hub.

Reply:
I should have been clear that one of our simplifying assumptions for the initial implementation is that the `budget` just represents the funds allocated to the ledger channel with the `hub`.

Reply:
For the time being (to ship web3torrent):
1. There will be at most one hub per site
2. The site will be responsible for providing the hub address
3. The budget will represent the funds in a single ledger channel with this hub

This means that, for the time being, it makes sense to include the `hubAddress` in the budget request.